doneill/er-cli-go
# Structure user response

Structure user response in markdown table format

```
| USERNAME |        EMAIL         | FIRST NAME | LAST NAME |                  ID                  | PIN  |              SUBJECT ID              |
|----------|----------------------|------------|-----------|--------------------------------------|------|--------------------------------------|
| cccy     |                      | CC         |  CY       | 015945ff-c220-4674-a070-3f1112e445fg |      | 12c245f6-8d77-4e15-a82c-be4a717034df |
```

**Test**
- Build the app
- Run `./er user`
- Verify the return display
